ui/slider.js

```diff
@@ -157,6 +157,48 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		}
 	},
 
+	_cleanValues: function( index, newValue ) {
+		var values,
+			i = 0 ;
+
+		if ( $.isArray( arguments[ 0 ] ) ) {
+			values = arguments[ 0 ] || this.options.values;
```

Comment:
How would this ever be `this.options.values` since `arguments[ 0 ]` is guaranteed to be an array?